en/browser-object-model-bom/cookies.md

```diff
@@ -18,19 +18,19 @@ book = Learn Javascript
 The `document.cookie` property is used to create, read and delete cookies. Creating cookie is pretty easy you need to provide the name and value
 
 ```javascript
-document.cookie = "book=Learn Javacript";
+document.cookie = "book=Learn Javascript";
```

Comment:
If possible can you rename this `Javascript` to `JavaScript`?

Reply:
sure why not?